roles/acme/meta/main.yml

```diff
@@ -13,3 +13,7 @@ galaxy_info:
     - letsencrypt
     - security
     - ssl
+  platforms:
```

Comment:
I think this one is a fluke. Why did you add this?

Reply:
Because ansible-lint expects it. However I now changed it to a existing value.